cmd/server/inventory.go

```diff
@@ -96,7 +96,7 @@ func listBuilders() error {
 
 func printConfigValidator(v pkgadapter.ConfigValidator) {
 
-	fmt.Printf("Params: \n")
+	fmt.Print("Params: \n")
```

Comment:
Then should we not use glog in the main.go for server (https://github.com/istio/mixer/blob/master/cmd/server/main.go#L21)?

We probably want consistency there. And, we should some mention of the exception (with reasoning) in our [docs](https://github.com/istio/mixer/blob/master/doc/dev/conventions.md#logging-conventions).